spec/formatting.md

```diff
@@ -106,16 +112,45 @@ In a _pattern_, the resolved value of an _expression_ or _markup_ is used in its
 The form that resolved values take is implementation-dependent,
 and different implementations MAY choose to perform different levels of resolution.
 
-> For example, the resolved value of the _expression_ `{|0.40| :number style=percent}`
-> could be an object such as
+> While this specification does not require it,
+> a _resolved value_ could be implemented by requiring each function implementation to
+> return a value matching the following interface:
 >
+> ```ts
+> interface MessageValue {
```

Comment:
Overall the direction seems OK.
But I am uneasy with specifying this interface.
Because:
- in "intertwines" even more the formatters and selectors (which I always maintained are different concepts, with different interfaces)
- it is just specified, without explaining how to use it, how it works.

So it feels over-specified.